packages/mobile/src/pincode/Pincode.tsx

```diff
@@ -0,0 +1,90 @@
+import Button, { BtnTypes } from '@celo/react-components/components/Button'
+import HorizontalLine from '@celo/react-components/components/HorizontalLine'
+import NumberKeypad from '@celo/react-components/components/NumberKeypad'
+import { fontStyles } from '@celo/react-components/styles/fonts'
+import { componentStyles } from '@celo/react-components/styles/styles'
+import React, { useCallback, useMemo } from 'react'
+import { ScrollView, StyleSheet, Text, View } from 'react-native'
+import PincodeTextbox from 'src/pincode/PincodeTextbox'
+
+interface Props {
+  onPress: () => void
+  title: string
+  placeholder: string
+  buttonText: string
+  maxLength?: number
+  isPinValid: (pin: string) => boolean
+  pin: string
+  onChangePin: (pin: string) => void
+}
+
+function Pincode(props: Props) {
```

Comment:
Consider adding a comment here about the diff btwn this file, PincodeEnter, and PincodeLock. I had to look around for a bit to figure it out, wasn't immediately obvious

Reply:
✅